src/gleam/http/cookie.gleam

```diff
@@ -0,0 +1,116 @@
+import gleam/int
+import gleam/list
+import gleam/option.{Option, Some}
+import gleam/string
+
+const epoch = "aa"
+
+pub type SameSitePolicy {
+  Lax
+  Strict
+  None
+}
+
+pub type Attributes {
+  Attributes(
+    // Expires is deprecated, we can still serialize a value for max compatibility
+    max_age: Option(Int),
+    domain: Option(String),
+    path: Option(String),
+    secure: Bool,
+    http_only: Bool,
+    // TODO I don't think it needs Option
+    same_site: Option(SameSitePolicy),
+  )
+}
+
+pub fn empty_attributes() {
+  Attributes(
+    max_age: option.None,
+    domain: option.None,
+    path: option.None,
+    secure: False,
+    http_only: False,
+    same_site: option.None,
+  )
+}
+
+pub fn default_attributes() {
+  Attributes(
+    max_age: option.None,
+    domain: option.None,
+    path: Some("/"),
+    secure: False,
```

Comment:
Plug sets secure true automatically if request/conn is https
https://github.com/elixir-plug/plug/blob/v1.10.3/lib/plug/conn.ex#L1464

This is not something that can be done without parsing the request as an argument.
Options as follows:

1. always set the secure attribute to True, this makes local development harder as cookies won't be sent to http://localhost
2. Make secure an arguement to this function. Which rather negates the value in a default_attributes function
3. Don't worry about it and leave security by default to a higher/framework level

Reply:
3 sounds good, though maybe we should have a note in the documentation saying that it ought to be set?

Reply:
I was thinking that, but then didn't because I was worried i'd just explain them all and slightly less well than just googling would give the reader. Happy to add a note if you think worth while

Reply:
I would like to at least hint to the user that there is something to google.

Just had a thought, what if this function took a `http.Scheme` and set `secure: True` if it's `Https`, would that make sense?

Reply:
There are reasons why you might want to do something different, but it makes some sense as a default.

Reply:
To take an http.Scheme as an argument makes a circular dependency `gleam/http` <-> `gleam/http/cookie`

I have just added a note instead